testsuite/MDAnalysisTests/coordinates/test_lammps.py

```diff
@@ -434,7 +434,8 @@ def u(self, tmpdir, request):
                 with gzip.GzipFile(f, 'wb') as fout:
                     fout.write(data)
 
-        yield mda.Universe(f, format='LAMMPSDUMP')
+        yield mda.Universe(f, format='LAMMPSDUMP',
+                           coordinate_convention='guess')
```

Comment:
I'd call this lammps coordinate convention to make it clear it's a format specific option?

Reply:
Roger roger